edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.adoc

```diff
@@ -12,3 +12,8 @@ You can do this at install time.
 include::modules/cnf-image-based-upgrade-shared-container-partition.adoc[leveloffset=+1]
 
 include::modules/ztp-image-based-upgrade-shared-container-partition.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+xref:../../../installing/install_config/installing-customizing.adoc#installation-special-config-butane-install_installing-customizing[Installing Butane]
```

Comment:
This just needs to be a list item per the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections):
```suggestion
* xref:../../../installing/install_config/installing-customizing.adoc#installation-special-config-butane-install_installing-customizing[Installing Butane]
```